toolchain/strace.py

```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/env python
+
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+#
+# See LICENSE for more details.
+#
+# Copyright: 2021 IBM
+# Author: Sachin Sant <sachinp@linux.vnet.ibm.com>
+
+import os
+import re
+
+from avocado import Test
+from avocado.utils import build, process, git
+from avocado.utils.software_manager import SoftwareManager
+
+
+class Strace(Test):
+
+    """
+    Strace is a diagnostic and debugging command line utility for Linux
+    :avocado: tags=os,testsuite
+    """
+
+    def setUp(self):
+        """
+        Build strace
+
+        Source:
+        http://github.com/strace/strace.git
+        """
+        smm = SoftwareManager()
+        packages = ['make', 'gcc', 'autoconf', 'automake']
+        for package in packages:
```

Comment:
can use package list directory here like 

`for package in ['make', 'gcc', 'autoconf', 'automake']:`  as it will help for garbage collector management well

Reply:
will make this change.